Misc/ACKS

```diff
@@ -1874,3 +1874,4 @@ Edison Abahurire
 Geoff Shannon
 Batuhan Taskaya
 Aleksandr Balezin
+Flavian Hautbois
```

Comment:
Place your name at appropriate place according to dictionary order (between "Hauser" and "Hawkes").